MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -161,18 +176,30 @@ public void AttachFileToLane (WebServiceLogin login, int lane_id, int lanefile_i
 		public void DeattachFileFromLane (WebServiceLogin login, int lane_id, int lanefile_id)
 		{
 			using (DB db = new DB ()) {
-				VerifyUserInRole (db, login, Roles.Administrator);
-				db.Audit (login, "WebServices.DeattachFileFromLane (lane_id: {0}, lanefile_id: {1})", lane_id, lanefile_id);
+				var lane = DBLane_Extensions.Create (db, lane_id);
+				VerifyUserInRoles (db, login, lane.additional_roles, false);
```

Comment:
VerifyUserInRoles automatically adds Administrator to the `additional_roles` to the list.